README.md

```diff
@@ -1,11 +1,12 @@
 <p align="center">
     <img src="https://github.com/sweepai/sweep/assets/26889185/39d500fc-9276-402c-9ec7-3e61f57ad233">
 </p>
-<p align="center">
-    <i>Bug Reports & Feature Requests ⟶&nbsp; Code Changes</i>
-</p>
-
-<p align="center">
+<a href="https://sweep.dev">
```

Comment:
replace this one with 

<p align="center">
    <i>Bug Reports & Feature Requests ⟶&nbsp; Code Changes</i>
</p>